runner/repl.js

```diff
@@ -117,6 +117,8 @@ const run = (source, _context, _filename, callback, run = true) => {
 
 const replServer = repl.start({ prompt: '> ', eval: run });
 
+replServer.on('reset', () => prev = "");
```

Comment:
Unrelated?

Reply:
Yeah, it's unrelated but also a good change.